appvm-scripts/usrbin/qubes-session

```diff
@@ -20,55 +20,49 @@
 #
 #
 
+# shellcheck disable=SC2164
+# shellcheck disable=SC1090
+# shellcheck disable=SC1091
+. /usr/lib/qubes/init/functions
+
+if qsvc guivm-gui-agent; then
+    if [ -e "$HOME/.xinitrc" ]; then
+        . "$HOME/.xinitrc"
+    fi
+    if [ -z "$XSESSION" ]; then
+        XSESSION=/usr/bin/xfce4-session
+    fi
+    /usr/bin/xfce4-session
```

Comment:
The script needs to terminate here (either use `exit` after or `exec` this one), otherwise you'd start the actual qubes-session when one logouts from xfce.
Also, this should use `$XSESSION`